generators/client-2/templates/src/main/webapp/app/_app.ng2module.ts

```diff
@@ -12,14 +13,30 @@ import { XSRFStrategyProvider } from './shared/XSRF-strategy.provider';
 import { HomeComponent } from './home/home.component';
 import { NavbarComponent } from './layouts/navbar/navbar.component';
 import { FooterComponent } from './layouts/footer/footer.component';
+import { ErrorComponent } from './layouts/error/error.component';
 <%_ if (enableTranslation){ _%>
 import { ActiveMenuDirective } from './layouts/navbar/active-menu.directive';
 <%_ } _%>
 
+import { appState } from './app.state';
+import { homeState } from './home/home.state';
+import { errorState, accessdeniedState } from './layouts/error/error.state';
+
+let routerConfig = {
+    otherwise: '/',
+    states: [
+        appState,
+        homeState,
+        errorState,
+        accessdeniedState
+    ]
+};
+
 @NgModule({
     imports: [
         BrowserModule,
         Ng1ToNg2Module,
+        UIRouterModule.forChild(routerConfig),
```

Comment:
@deepu105 forRoot provides a new instance of UIRouter object.  It should not be used when using ng1-to-ng2 because the instance is already provided by angular 1 and Ng1ToNg2Module. Once The app is pure ng2 then you need forRoot.